docs/sample_config.yaml

```diff
@@ -278,6 +278,23 @@ listeners:
 # Used by phonehome stats to group together related servers.
 #server_context: context
 
+# Resource-constrained Homeserver Settings
+#
+# If limit_remote_rooms.enabled is True, the room complexity will be
+# checked before a user joins a new remote room. If it is above
+# limit_remote_rooms.complexity, it will disallow joining or
+# instantly leave.
```

Comment:
Do we have any docs on what these values mean? It would be good to give some form of hand wavey guidance on what people can set this to, since we're making this config option public.

Reply:
1 == 500 state events but that kinda means nothing as well because lol :( it is basically a black box number and there's no "good" setting without looking at the complexity of the rooms you want to join and setting it higher than that

Reply:
@hawkowl How does one determine the complexity of a given room?

Also, I think it would be a good idea to document a few tested values for a few configurations, fe. one for very small HS, one for a medium one, one for a big one, but still not big enough to not be wary of a possible failure. You know, some base for quick set up and further optimization.